.gitignore

```diff
@@ -99,6 +99,7 @@ UnitTesting/src/**/*
 UnitTesting/tests/*_test_results.xml
 UnitTesting/tests/csharp/Assets/Tests/generated/**/*
 UnitTesting/tests/javascript/tests.js
+Tests/assets/test_account_creds.txt
```

Comment:
I don't think we need test_account_creds inside of the assets folder. It should exist next to the test executable

Reply:
It doesn't compile otherwise?  It tried to check it in, hence the gitignore entry

Reply:
All good